pom.xml

```diff
@@ -3,12 +3,13 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>3.50</version>
+		<version>4.50</version>
+		<relativePath />
 	</parent>
 
   <properties>
-    <jenkins.version>1.653</jenkins.version>
-    <java.level>8</java.level>
+    <jenkins.version>2.346.3</jenkins.version>
+    <java.level>11</java.level>

```

Comment:
Is that possible to make the baseline version upgrade decoupled with open-telemetry related changes? 
This is a good feature and legacy Jenkins version should be able to use it as well.

Reply:
@cashlalala 
I meet a lot of dependency errors when I use the original baseline version.
I believe this is introduced by the opentelemetry plugin. In other words, to use it, users must upgrade their Jenkins to meet the requirements.
I use the same baseline version as the opentelemetry ones.

Any idea?

Reply:
Please see #85

Reply:
I was thinking that you might find the preivous version of open telemetry plugin with compatibility of jenkins 1.653 
then upgrade the baseline for newer master and open telemetry plugin.

Reply:
@cashlalala 
Please check https://github.com/jenkinsci/opentelemetry-plugin/blob/105ad4bf8da347dbc26bb42e55c97404fdef2884/pom.xml#L22C1-L23C1
It seems opentelemetry chose [2.235.1](https://www.jenkins.io/changelog-stable/#v2.235.1) from the beginning.

Reply:
I may redo the change. 
Perhaps I can get rid of the opentelemetry plugin completely!